compose-builder/Makefile

```diff
@@ -64,17 +65,98 @@ endif
 ifeq (ds-virtual, $(filter ds-virtual,$(ARGS)))
 	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-device-virtual.yml
 endif
-# Add/switch to use MQTT Message Bus
+ifeq (asc-http, $(filter asc-http,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-asc-http-export.yml
+endif
+
+# Add ModBus Simulator
+ifeq (modbus-sim, $(filter modbus-sim,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-modbus-simulator.yml
+endif
+
+# Add Application Services
+ifeq (asc-http, $(filter asc-http,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-asc-http-export.yml
+endif
+ifeq (asc-http-s, $(filter asc-http-s,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-asc-http-export-secure.yml
+endif
+ifeq (asc-mqtt, $(filter asc-mqtt,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-asc-mqtt-export.yml
+endif
+ifeq (asc-mqtt-s, $(filter asc-mqtt-s,$(ARGS)))
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-asc-mqtt-export-secure.yml
+endif
+
+# Add switch to use MQTT Message Bus
 ifeq (mqtt, $(filter mqtt,$(ARGS)))
-	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-mqtt-messagebus.yml
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-mqtt-messagebus.yml -f add-mqtt-broker.yml
 	MQTT:=-mqtt
 endif
+
 # Add Security services and settings
 ifeq (no-secty, $(filter no-secty,$(ARGS)))
 	NO_SECURITY:=-no-secty
 else
 	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-security.yml
 endif
+
+# Build compose for TAF secure testing (ignore all other compose file options)
+ifeq (taf-secty, $(filter taf-secty,$(ARGS)))
+	COMPOSE_FILES:=-f docker-compose-base.yml \
+				   -f add-security.yml \
+				   -f add-taf-app-services.yml \
+				   -f add-asc-http-export-secure.yml \
+				   -f add-asc-http-export.yml \
```

Comment:
The app-service for http-export profiles did not use on performance metrics test.